org.adempiere.base/src/org/compiere/model/MUser.java

```diff
@@ -796,7 +798,7 @@ public boolean isNotificationNote()
 		// are found but also roles which delegate org access to the user level where
 		// this user has access to the org in question
 		String sql = "SELECT * FROM AD_Role r " 
-			+ "WHERE r.IsActive='Y'" 
+			+ "WHERE r.IsActive='Y' AND r.AD_Client_ID=? " 
```

Comment:
This change breaks the MUser.isAdministrator validation

Reply:
so a user is still an administration even when the user is logging in as "GardenWorld User" role ? That doesn't sounds quite right to me.

Reply:
Yes, users with System role are administrators when logged in any tenant - they have certain powers (and those are required) - for example change a password without knowing the previous.

Reply:
ok, that's a trivial fix, I will change that.